data-products-dp0-3/table-access-and-queries.rst

```diff
@@ -117,6 +118,38 @@ The N:1 nature of these joins means that the data retrieved can contain columns
 be larger than expected, and take a long time to execute.
 
 
+.. _DP0-3-Table-Access-ADQL-passing-list:
+
+Advice for passing an existing list to a new query
```

Comment:
Could we shorten? E.g., to "Passing a list to a query" or "List-based query" or "Query a list of objects", or similar.